src/main/java/mate/academy/rickandmorty/service/CharacterServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.rickandmorty.service;
+
+import java.util.List;
+import java.util.Random;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.external.CharacterDataDto;
+import mate.academy.rickandmorty.dto.internal.CharacterDto;
+import mate.academy.rickandmorty.mapper.CharacterMapper;
+import mate.academy.rickandmorty.repository.CharacterRepository;
+import org.springframework.stereotype.Service;
+
+@RequiredArgsConstructor
+@Service
+public class CharacterServiceImpl implements CharacterService {
+    private final CharacterRepository repository;
+    private final CharacterMapper mapper;
+    private final Long maxId = 257L;
+
+    @Override
+    public CharacterDto getRandomCharacter() {
+        long generatedRandomId = new Random().nextLong(1, maxId);
```

Comment:
1. what if we'd have more than 257 characters? 
2. random should be `private final` class-level field
```suggestion
        long size = characterRepository.count();
        long generatedRandomId = random.nextLong(1, size);
```

Reply:
@kerrrusha, 
1) if I count size in getRandomCharacter method it would be requested DB 2 times per every request. Is it ok? First I wrote this in case data would be modified/deleted:
` private Long getMaxId() {
        return repository.findAll().stream()
                .map(CustomCharacter::getId)
                .sorted(Comparator.reverseOrder())
                .findFirst()
                .get();
    }`   
2) Ok.